src/events.h

```diff
@@ -49,4 +52,19 @@ void handle_state(monitor_t *m, desktop_t *d, node_t *n, xcb_atom_t state, unsig
 void mapping_notify(xcb_generic_event_t *evt);
 void process_error(xcb_generic_event_t *evt);
 
+// These are used when handling client movement requests;
+// they are directly from https://specifications.freedesktop.org/wm-spec/wm-spec-latest.html#idm46035372579808
+#define _NET_WM_MOVERESIZE_SIZE_TOPLEFT      0
+#define _NET_WM_MOVERESIZE_SIZE_TOP          1
+#define _NET_WM_MOVERESIZE_SIZE_TOPRIGHT     2
+#define _NET_WM_MOVERESIZE_SIZE_RIGHT        3
+#define _NET_WM_MOVERESIZE_SIZE_BOTTOMRIGHT  4
+#define _NET_WM_MOVERESIZE_SIZE_BOTTOM       5
+#define _NET_WM_MOVERESIZE_SIZE_BOTTOMLEFT   6
+#define _NET_WM_MOVERESIZE_SIZE_LEFT         7
+#define _NET_WM_MOVERESIZE_MOVE              8   /* movement only */
+#define _NET_WM_MOVERESIZE_SIZE_KEYBOARD     9   /* size via keyboard */
+#define _NET_WM_MOVERESIZE_MOVE_KEYBOARD    10   /* move via keyboard */
+#define _NET_WM_MOVERESIZE_CANCEL           11   /* cancel operation */
```

Comment:
These constants are already defined in `xcb_ewmh.h`.

Reply:
Thank you, I updated the changes in d7a31ad. I don't know if github can do a PR as a squashed merge but I'm fine re-committing the 3 changes as 1 because it makes the history cleaner.